HHS/Head-Start-TTADP
# Display external link warning

**Description of change**
When a user clicks on a external link (external is defined as anything outside the current hostname), we should show a modal telling the user about the security risks.

**How to test**
pull down changes.
On Topics & Resources page, enter external URL's
When reviewing the page, you should be able to click that section and see the links as hyper links.
Clicking on them will make a modal show up.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/177

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
